packages/shared/src/modules/quiz/libs/enums/score-validation-message.enum.ts

```diff
@@ -0,0 +1,11 @@
+import { ScoreValidationRule } from "./score-validation-rule.enum.js";
+
+const ScoreValidationMessage = {
+	INVALID_REQUEST: "Invalid request.",
+	ITEMS_MIN_LENGTH: `At least ${String(ScoreValidationRule.ITEMS_MIN_LENGTH)} item to update must be provided.`,
+	SCORE_MAX_VALUE: `Score must be at most ${String(ScoreValidationRule.SCORE_MAX_VALUE)}.`,
```

Comment:
Explicit cast to String seems redundant here because the cast will be done by the string template (`${}`) itself